### ISSUE TITLE
(Co)-Maintainers wanted

### ISSUE THREAD

**Issue by Lecrapouille** [Lecrapouille/gdcef#62]
I'm giving up with this project! If people are interested to maintain this project ping me :)

**Comment by pimhakkert** [comment]
I'm interested in adding 2 new features and maybe maintaining a little bit of the code if I have time:
- Transparency of the browser view. My use ase of this library is to add a UI to my game and I think there needs to be an open source alternative to something like Coherent's GameFace, even if it is not an optimal solution
- Extending on my previous point, allowing for fast click throughput. If you make the browser view the size of the entire game screen, you'll need to be able to let the player click on game objects underneath the browser view.

If needs be and I'm able to handle it, I'll fix a few bugs occasionally as well.
@Lecrapouille I'm interested to hear what you think of this.

**Comment by Lecrapouille** [comment]
@pimhakkert Excellent!

**Comment by Lecrapouille** [comment]
I let you make create your initial pull requests and if I consider the quality okay I'll give you them developer rights to this repo.

**Comment by pimhakkert** [comment]
Thank you. I will get to work on something then :)

**Comment by Lecrapouille** [comment]
@eliopark-nsuslab are you interested? You have a pending invitation to accept

**Comment by Lecrapouille** [comment]
Offer still available for people interested.

**Comment by yonacwy** [comment]
Im working on getting it running in blazium, there where issues with build paths and now some runtime errors.

It would be cool to just compile it within blazium/jenova

![Image](https://github.com/user-attachments/assets/b5c9acb0-bd7a-48ed-a5cf-c5c93697d33b)

**Comment by Lecrapouille** [comment]
@yonacwy Thank you for the report ! Do you want to be maintainer or you are opening a ticket ? In the last case, pleas open a new ticket.

**Comment by yonacwy** [comment]
yeah i can help, those where user errors issues resolved when compiled from source.

**Comment by Lecrapouille** [comment]
Ok pull requests are welcome.
